### ISSUE TITLE
CSS import error

### ISSUE THREAD

**Issue by zoechi** [dart-lang/polymer-dart#30]
I can't get rid of this warning

```
Failed to inline stylesheet: Could not find asset bwu_datagrid|example/src/composite_editor_item_details/asset/examples.css. null (more details)
example/composite_editor_item_details.html:405:5
<link rel="stylesheet" href="src/composite_editor_item_details/asset/examples.css">
```

It is this import
https://github.com/bwu-dart/bwu_datagrid/blob/master/example/src/composite_editor_item_details/composite_editor_view.html#L10

This is the entry page

https://github.com/bwu-dart/bwu_datagrid/blob/master/example/composite_editor_item_details.html
the app-element is https://github.com/bwu-dart/bwu_datagrid/blob/master/example/src/composite_editor_item_details/app_element.html

I tried adding a few `../` but I wasn't able to find a way to point it to the correct directory.
The same import works in all other examples. The difference here is that the import is in an element `composite-editor-view` which is imported by `app-element`. In the other examples the `examples.css` is only used in the app-element.

**Comment by jakemac53** [comment]
I think this is possibly because of the `asset` folder in the path?

**Comment by zoechi** [comment]
What do you mean with "`asset` folder in the path"?

**Comment by jakemac53** [comment]
Nvm, that wasn't the issue :). I have narrowed this down the the import inliner, its definitely just a bug.

**Comment by jakemac53** [comment]
https://codereview.chromium.org/993423004/ is out to fix this, thanks for the report! This looks like a pretty nasty one.

**Comment by zoechi** [comment]
Thanks a lot for the fast fix!

**Comment by jakemac53** [comment]
this is published now, web_components 0.10.5+3
